docs/_sass/_main.scss

```diff
@@ -562,6 +562,37 @@ button {
                 margin-left: 12px;
             }
         }
+
+        .custom-video {
```

Comment:
NAB: I feel like we can just call this `.help-video` - something generic instead of custom video.

Reply:
maybe just `video`?